pkg/netceptor/ping_test.go

```diff
@@ -0,0 +1,251 @@
+package netceptor_test
+
+import (
+	"context"
+	"errors"
+	"net"
+	"testing"
+	"time"
+
+	"github.com/ansible/receptor/pkg/netceptor"
+	"github.com/ansible/receptor/pkg/netceptor/mock_netceptor"
+	"github.com/golang/mock/gomock"
+)
+
+// setupTest sets up TestPing tests.
+func setupTest(t *testing.T) (*gomock.Controller, *mock_netceptor.MockNetcForPing, *mock_netceptor.MockPacketConner) {
+	ctrl := gomock.NewController(t)
+
+	// Prepare mocks
+	mockNetceptor := mock_netceptor.NewMockNetcForPing(ctrl)
+	mockPacketConn := mock_netceptor.NewMockPacketConner(ctrl)
+
+	mockPacketConn.EXPECT().SetHopsToLive(gomock.Any()).AnyTimes()
```

Comment:
Interesting that you have to set the times to `Any` and `0` in the set ups and tear downs, any reason for this?